cishouseholds/pipeline/mapping.py

```diff
@@ -11,6 +11,24 @@
     "7 times or more": 7,
 }
 
+consolidation_categories = {
+    "box_empty": 1,
+    "box_multipacked": 2,
+    "exception_undefined": 3,
+    "kpa_overpacked": 4,
+    "kpa_overpacked_different_ref_different_type": 5,
+    "kpa_overpacked_different_ref_same_type": 6,
+    "kpa_unboxed": 7,
+    "kpa_unscanned": 8,
+    "sample_bagged_ziplock": 9,
```

Comment:
double checking against supplier doc, this is 'sample_bagged_zip' instead of 'sample_bagged_ziplock'